app/assets/javascripts/admin/modules/document-history-paginator.js

```diff
@@ -0,0 +1,55 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function DocumentHistoryPaginator (module) {
+    this.module = module
+  }
+
+  DocumentHistoryPaginator.prototype.init = function () {
+    var fetch = window.fetch
+    if (!fetch) { return }
+
+    this.setupEventListeners()
+  }
+
+  DocumentHistoryPaginator.prototype.setupEventListeners = function () {
+    var module = this.module
+    var newerLink = module.querySelector('.app-view-document-history-tab__newer-pagination-link')
+    var olderLink = module.querySelector('.app-view-document-history-tab__older-pagination-link')
+
+    if (newerLink) {
```

Comment:
Since this is really the same code twice you could define a setupLink function and the call it with `newerLink` and `olderLink`

Reply:
Thanks for the review. Great shout. I'll sort it before merge